res/css/views/settings/tabs/_SecurityRoomSettingsTab.scss

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2019 New Vector Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.mx_SecurityRoomSettingsTab label {
+  display: block;
```

Comment:
Minor style nit: Use 4 spaces for indentation.

Reply:
I'm going to fix all of the CSS I've added for settings in one go because they're all at 2 spaces.